tools/common-code/cmonitor_launcher.py

```diff
@@ -0,0 +1,195 @@
+# =======================================================================================================
```

Comment:
this file (cmonitor_launcher.py) really deserve its own folder: tools/launcher.. tools/common-code is sort of "library" folder where we store Python code that is "importable" and usable as standalone library.

Reply:
another cosmetic note: make sure to run "black" on this file to ensure consistent formatting

Reply:
we also need the python3 shebang:

```
#!/usr/bin/python3
```

as very first line